app/javascript/searchableItemList/searchableItemList.js

```diff
@@ -75,12 +75,16 @@ export function performInitialSearch({ searchOptions = {} }) {
   const responsePromise = fetchSearch('reactions', dataHash);
   return responsePromise.then((response) => {
     const reactions = response.result;
+    const availableTags = [
+      ...new Set(reactions.flatMap((rxn) => rxn.reactable.tag_list)),
+    ];
```

Comment:
I'm requesting this change after I approved, sorry Andy: I think we should call `.sort()` here so that tags will always be sorted alphabetically and not depend on the order of the array of reading list items